src/Symfony/Component/Messenger/Tests/EventListener/ResetServicesListenerTest.php

```diff
@@ -31,7 +32,8 @@ public static function provideResetServices(): iterable
      */
     public function testResetServices(bool $shouldReset)
     {
-        $servicesResetter = $this->createMock(ServicesResetter::class);
+        /** @var ServicesResetterInterface&MockObject $servicesResetter */
```

Comment:
this `@var` is not necessary. `$this->createMock` already has this as return type.